packages/talker_flutter/lib/src/controller/talker_screen_controller.dart

```diff
@@ -69,7 +70,7 @@ class TalkerScreenController extends ChangeNotifier {
   Future<String> saveLogsInFile(String logs) async {
     final dir = await getTemporaryDirectory();
     final dirPath = dir.path;
-    final file = await File('$dirPath/talker_logs_${DateTime.now()}.txt')
+    final file = await File(join(dirPath, 'talker_logs_${DateTime.now().toString().replaceAll(":", " ")}.txt'))
```

Comment:
I think that depend on path library only for this method is too much for our package